### PR TITLE
Adapt the package for SUSE Manager 4.3

### PR DESCRIPTION
**Pull request by juliogonzalez**
I also adapted the SPEC to the most recent SLE/openSUSE format guidelines.

### CODE REVIEW

File: package/skelcd-control-suse-manager-server.spec [hunk 1]
```diff
@@ -38,8 +38,8 @@ BuildRequires:  libxml2-tools
 BuildRequires:  yast2-installation-control >= 4.1.5
 
 # Original SLES control file (skip registration)
-BuildRequires: skelcd-control-SLES >= 15.2.0
-BuildRequires: diffutils
+BuildRequires:  diffutils
+BuildRequires:  skelcd-control-SLES >= 15.2.0
```
Comment (juliogonzalez) [inline package/skelcd-control-suse-manager-server.spec:42]: Do I need to change this from `15.2.0` to `15.4.0`? Even changing it, the CI failed with the same message.

Comment (jreidinger) [reply package/skelcd-control-suse-manager-server.spec:42]: @lslezak can you check ci failure? it looks like in our repos it is missing some migration product.

Comment (juliogonzalez) [reply package/skelcd-control-suse-manager-server.spec:42]: Same problem seems to be happening at https://github.com/yast/skelcd-control-suse-manager-proxy/pull/16

Comment (juliogonzalez) [reply package/skelcd-control-suse-manager-server.spec:42]: Any news? :-)

Comment (juliogonzalez) [reply package/skelcd-control-suse-manager-server.spec:42]: Can we speedup this somehow? Is there any way I can help? @lslezak @jreidinger ?

**Review by jreidinger — APPROVED** [review]
as CI failure looks unrelated, lets approve it

**Comment by lslezak** [comment]
I have added the missing package several days ago, it was just enough to restart the CI to get it green.

**Pull request merged**